captum/attr/_utils/summarizer.py

```diff
@@ -140,14 +140,14 @@ def _reorder_stats(stats):
     return stats, summary_stat_indexs
 
 
-def CommonSummarizer():
+def CommonSummarizer(summarizer_class=Summarizer):
     r"""
     Returns a summarizer with common summary statistics, specifically with:
         Mean, Sample Variance, Sample Std Dev, Min, Max
     """
     from captum.attr._utils.stat import Mean, Var, StdDev, Min, Max
```

Comment:
It looks like the summarizer has the imports everywhere inside the classes and functions. To be consistent with other code can we, please, move the imports to the beginning of the file, @miguelmartin75  ?